File: exercice2/furniture.ts
class Furniture {
  editor: string;
  designer: string;
  year: number;

  constructor(editor: string, designer: string, year: number) {
    this.editor = editor;
    this.designer = designer;
    this.year = year;
  }
}

let wassily: Furniture = new Furniture("Knoll", "Marcel Breuer", 1925);
console.log(wassily.editor, wassily.designer, wassily.year);

wassily.year = 1926;
console.log(wassily.year) 

File: exercice2/furniture.java
class Playground {
    public static void main(String[ ] args) {
        Furniture wassily = new Furniture ("Knoll", "Marcel Breuer", 1925);
        System.out.print(wassily.editor + " " + wassily.designer + " " + wassily.year);
    }
}

public class Furniture {
    String editor;
    String designer;
    int year;

    public Furniture(String editor, String designer, int year){
      this.editor = editor;
      this.designer = designer;
      this.year = year;
    }
}
